index.html

```diff
@@ -793,7 +1027,81 @@ <h3>
                   </li>
                 </ol>
               </li>
-            </ol>
+            </ol></del> <ins cite="#a15">
+            <ol>
+              <li>Set |permission| to [=request geolocation permission=] for
+              |descriptor|.
+              </li>
+              <li>Let |handle permission denial| be the following steps:
+                <ol>
+                  <li>If |watchId| was passed, [=list/remove=] |watchId| from
+                  |watchIDs|.
+                  </li>
+                  <li>[=Call back with error=] passing |errorCallback| and
+                  {{GeolocationPositionError/PERMISSION_DENIED}}.
+                  </li>
+                  <li>Terminate this algorithm.
```

Comment:
It seems unclear whether "terminate this algorithm" is returning from the |handle permission denial| steps or from the request position algorithm (as intended). I suggest removing it here and changing the callers to say "run |handle permission denial| and terminate this algorithm".